README.md

```diff
@@ -30,6 +30,7 @@ dataset.upload_dataset(task="text_clf", split="train", module_dir="path_to_imdb_
 
 ## Notebooks
 
+ - [Basics](basics/basics.ipynb)
```

Comment:
Maybe expanding a bit on what to expect in these notebooks helps to user

Reply:
Its there in the beginning of notebooks. Except for basics, other notebooks(like `dataset_upload`) would have precise naming. So not needed I think